### PR TITLE
Chloe Price

### PR DESCRIPTION
**Pull request by chloell5**
Here is my prework! Let me know if there are any questions!

### CODE REVIEW

File: final_prep/mod_zero_hero.rb [hunk 1]
```diff
@@ -1,84 +1,154 @@
...
+#   - energy_level = 50
 
 # - Create the following class methods
 #   - say_name, should print the hero's name to the terminal
 #   - maximize_energy, should update the energy_level to 1000
 #   - gain_power, should take an argument of a number and INCREASE the power_level by that number
+class SuperHero
+  attr_accessor :name, :super_power, :age
+
+  def initialize
```
Comment (mschae16) [inline final_prep/mod_zero_hero.rb:110]: Where are you declaring your 3 dynamic arguments?  I think `name`, `super_power` and `age` need to be passed in as parameters on the `initialize` function.

File: final_prep/mod_zero_hero.rb [hunk 2]
```diff
@@ -1,84 +1,154 @@
...
+    energy_level = 1000
+  end
+
+  def gain_power(number)
+    @power_level = @power_level * number
+  end
+end
 
 # - Create 2 instances of your SuperHero class
+codeman = SuperHero.new
```
Comment (mschae16) [inline final_prep/mod_zero_hero.rb:134]: I think you want to create your `SuperHero` instance with 3 arguments passed in here, like so:

```
codeman = SuperHero.new('CodeMan', 'Debugging', 35)
```

**Review by mschae16 — APPROVED** [review]
@chloell5 - Nice job on the prework. You are officially `Turing Ready`. Let me know if you have any questions.

**Comment by chloell5** [comment]
I'll fiddle with the sections you highlighted, I want to make sure I have this figured out. Thank you!

**Pull request closed without merging**